diff --git a/config.def.h b/config.def.h
index bcca4d2..a094ecc 100644
--- a/config.def.h
+++ b/config.def.h
@@ -7,9 +7,9 @@ static const unsigned int gappiv    = 10;       /* vert inner gap between window
 static const unsigned int gappoh    = 10;       /* horiz outer gap between windows and screen edge */
 static const unsigned int gappov    = 10;       /* vert outer gap between windows and screen edge */
 static const int smartgaps          = 0;        /* 1 means no outer gap when there is only one window */
-static const float rootcolor[]      = {0.3, 0.3, 0.3, 1.0};
-static const float bordercolor[]    = {0.5, 0.5, 0.5, 1.0};
-static const float focuscolor[]     = {1.0, 0.0, 0.0, 1.0};
+static const float rootcolor[]      = {0.93, 0.93, 0.93, 1.0};
+static const float bordercolor[]    = {0.13, 0.13, 0.13, 1.0};
+static const float focuscolor[]     = {0.27, 0.27, 0.27, 1.0};
 
 /* tagging */
 static const char *tags[] = { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
