diff --git a/config.def.h b/config.def.h
index 646a4a4..b5f3c63 100644
--- a/config.def.h
+++ b/config.def.h
@@ -49,8 +49,8 @@ static const struct xkb_rule_names xkb_rules = {
 	.options = "",
 };
 
-static const int repeat_rate = 25;
-static const int repeat_delay = 600;
+static const int repeat_rate = 50;
+static const int repeat_delay = 150;
 
 /* Trackpad */
 static const int tap_to_click = 1;
