diff --git a/config.def.h b/config.def.h
index 5ff76ba..14ee2e8 100644
--- a/config.def.h
+++ b/config.def.h
@@ -18,8 +18,9 @@ static const Rule rules[] = {
 	/* app_id     title       tags mask     isfloating   monitor */
 	/* examples:
 	{ "Gimp",     NULL,       0,            1,           -1 },
-	*/
 	{ "firefox",  NULL,       1 << 8,       0,           -1 },
+	*/
+	{"",NULL,0,0,-1},
 };
 
 /* layout(s) */
