diff --git a/config.def.h b/config.def.h
index 7dfc2f7..7590df1 100644
--- a/config.def.h
+++ b/config.def.h
@@ -84,8 +84,6 @@ static const Key keys[] = {
 	{ MODKEY,                    XKB_KEY_l,          focusstack,     {.i = -1} },
 	{ MODKEY|WLR_MODIFIER_SHIFT, XKB_KEY_H,          movestack,      {.i = +1} },
 	{ MODKEY|WLR_MODIFIER_SHIFT, XKB_KEY_L,          movestack,      {.i = -1} },
-	{ MODKEY,                    XKB_KEY_i,          incnmaster,     {.i = +1} },
-	{ MODKEY,                    XKB_KEY_d,          incnmaster,     {.i = -1} },
 	{ MODKEY,                    XKB_KEY_k,          setmfact,       {.f = -0.05} },
 	{ MODKEY,                    XKB_KEY_j,          setmfact,       {.f = +0.05} },
 	{ MODKEY|WLR_MODIFIER_ALT,   XKB_KEY_l,          incgaps,        {.i = +1 } },
