diff --git a/config.def.h b/config.def.h
index 02483ed..de1a2d9 100644
--- a/config.def.h
+++ b/config.def.h
@@ -69,11 +69,12 @@ static const int natural_scrolling = 0;
 #define SHCMD(cmd) { .v = (const char*[]){ "/bin/sh", "-c", cmd, NULL } }
 
 /* commands */
-static const char *termcmd[] = { "alacritty", NULL };
-static const char *menucmd[] = { "wofi", "-S", "run", "-font", "Menlo For Powerline", NULL };
+static const char *termcmd[]      = { "alacritty", NULL };
+static const char *menucmd[]      = { "wofi", "-S", "run", "-font", "Menlo For Powerline", NULL };
 static const char *poweroffcmd[]  = { "sudo", "poweroff", NULL };
-static const char *firefoxcmd[]  = { "firefox", NULL };
+static const char *firefoxcmd[]   = { "firefox", NULL };
 static const char *chromiumcmd[]  = { "chromium", NULL };
+static const char *mutecmd[]      = { "pactl", "set-source-mute", "@DEFAULT_SOURCE@", "toggle", NULL };
 
 static const Key keys[] = {
 	/* Note that Shift changes certain key codes: c -> C, 2 -> at, etc. */
@@ -129,6 +130,7 @@ static const Key keys[] = {
 	{ MODKEY|WLR_MODIFIER_SHIFT|WLR_MODIFIER_CTRL, XKB_KEY_Delete, spawn, {.v = poweroffcmd}},
 	{ MODKEY|WLR_MODIFIER_SHIFT,                   XKB_KEY_F,      spawn, {.v = firefoxcmd}},
 	{ MODKEY|WLR_MODIFIER_SHIFT,                   XKB_KEY_C,      spawn, {.v = chromiumcmd}},
+	{ MODKEY,                                      XKB_KEY_a,      spawn, {.v = mutecmd}},
 
 	/* Ctrl-Alt-Backspace and Ctrl-Alt-Fx used to be handled by X server */
 	{ WLR_MODIFIER_CTRL|WLR_MODIFIER_ALT,XKB_KEY_Terminate_Server, quit, {0} },
